### DISCUSSION TITLE
UNO Platform - Build a Todo App - Part 1

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#86] [General]
# UNO Platform - Build a Todo App - Part 1

This is the beginning of a mini series shedding some lights on the UNO Platform.
The UNO Platform enables us to write code once and run it on a wide range of platforms (Windows, MacOS, Android, Linux, ...).
So let's have a look at this. To do so we will create a small Todo App Kanban-style so we can organize our thoughts and work ;)

https://steven-giesel.com/blogPost/b2234ada-0978-4c7b-841e-ca6a255247b0

**Comment by pishvaei** [comment]
thank you . It was very smooth

**Reply by linkdotnet** [reply]
Hey @pishvaei  - thanks for the kind words!

**Comment by DevTKSS** [comment]
I would really appreciate to learn from your provided learning material how to do that TODO app as there seem to be only a figma tutorial from uno itself, so could you consider to update this post to latest or at least single head uno app version which we do have now?

**Reply by linkdotnet** [reply]
Very good point given that this was done some years ago. Can't promise anything for the next weeks as I am bit swamped.

But maybe @mtmattei / @unoplatform can help out here. You probably have some really good ressources to get people started with the amazing **6.0** release.
